traffic_router/shared/pom.xml

```diff
@@ -113,8 +113,8 @@ under the License.
 		</dependency>
 		<dependency>
 			<groupId>org.slf4j</groupId>
-			<artifactId>slf4j-log4j12</artifactId>
-			<version>1.7.5</version>
+			<artifactId>slf4j-simple</artifactId>
+			<version>1.7.32</version>
```

Comment:
`slf4j-log4j12` should be replaced with `log4j-slf4j-impl`

Reply:
What's wrong with `slf4j-simple` ? why does it need to be replaced with `log4j-slf4j-impl`

Reply:
`log4j-slf4j-impl` is what to use for `log4j2`. From [https://logging.apache.org/log4j/2.x/log4j-slf4j-impl](https://logging.apache.org/log4j/2.x/log4j-slf4j-impl):

> The Log4j 2 SLF4J Binding allows applications coded to the SLF4J API to use Log4j 2 as the implementation.
> [...]
> log4j-slf4j-impl should be used with SLF4J 1.7.x releases or older.

Reply:
It's not breaking anything to leave it as-is, though. This doesn't hold up the PR, maybe just something to do in the future.